Pod/Classes/SKYContainer.h

```diff
@@ -61,33 +59,27 @@ typedef void (^SKYContainerUserOperationActionCompletion)(SKYUser *user, NSError
 + (nonnull SKYContainer *)defaultContainer;
 
 /// Undocumented
-@property (nonatomic, weak) id<SKYContainerDelegate> delegate;
+@property (nonatomic, readonly) SKYAuthContainer *auth;
```

Comment:
I think it is a good time to add nonnull to the new properties which return sub-containers.